token-handler/src/common/events.rs

```diff
@@ -0,0 +1,21 @@
+use transaction::OperationData;
+
+multiversx_sc::imports!();
+
+#[multiversx_sc::module]
+pub trait EventsModule {
+    #[event("executedBridgeOp")]
+    fn execute_bridge_operation_event(
+        &self,
+        #[indexed] hash_of_hashes: ManagedBuffer,
+        #[indexed] hash_of_bridge_op: ManagedBuffer,
+    );
+
+    #[event("deposit")]
+    fn deposit_event(
```

Comment:
This event should be in enshrine, token-handler will not emit this event (after transfer and execute by user)